src/calculateBullsAndCows.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
+'use strict';
+
+const calculateBullsAndCows = (randomNumber, guess) => {
+  const numbers = Array(10);
+
+  numbers.fill(0);
+
+  let cows = 0;
+  let bulls = 0;
+
+  for (let i = 0; i < guess.length; i++) {
+    if (randomNumber[i] === guess[i]) {
+      bulls++;
+    } else {
+      if (numbers[+guess[i]] > 0) {
+        cows++;
+      }
+
+      if (numbers[+randomNumber[i]] < 0) {
+        cows++;
+      }
+
+      numbers[+randomNumber[i]]++;
+      numbers[+guess[i]]--;
+    }
+  }
+
```

Comment:
I think this function may looks easier because we always generate different digits) So if it isn't a bull, we need only check if a secret includes guess[i] to find out if it a cow or no))